libr/core/cconfig.c

```diff
@@ -850,15 +850,16 @@ static int cb_cfgdebug(void *user, void *data) {
 			r_config_set (core->config, "asm.arch", "bf");
 		if (core->file) {
 #if __WINDOWS__
-			r_debug_select (core->dbg, core->dbg->pid,
-					core->dbg->tid);
+			r_debug_select (core->dbg, core->dbg->pid, core->dbg->tid);
 #else
-			r_debug_select (core->dbg, core->file->desc->fd,
-					core->file->desc->fd);
+			int pid = r_io_desc_get_pid (core->io, core->file->desc->fd);
+			r_debug_select (core->dbg, pid, pid);
 #endif
 		}
 	} else {
-		if (core->dbg) r_debug_use (core->dbg, NULL);
+		if (core->dbg) {
```

Comment:
not sure, i'd like to wait for the refactor of windbg, before touching this. The if WINDOWS must die